scripts/setupTypeScriptRollup.js

```diff
@@ -0,0 +1,135 @@
+// @ts-check
+
+/** This script modifies the project to support TS code in .svelte files like:
+  <script lang="ts">
+ 
+  As well as validating the code for CI.
+  */
+
+/**  To work on this script:
+  rm -rf test-template sapper-template && git clone sveltejs/sapper-template test-template && node scripts/setupTypeScript.js test-template
+*/
+
+const fs = require("fs")
+const path = require("path")
+const { argv } = require("process")
+
+const projectRoot = argv[2] || path.join(__dirname, "..")
+
+console.log("Adding typescript with rollup")
+
+if(fs.existsSync(path.join(projectRoot, "package.json"))) {
+
+  throw new Error("package.json already created. This script only works on plain new templates!")
```

Comment:
When you use `degit` to clone `sveltejs/sapper-template#rollup` it comes with `package.json` because `degit` created that from `package_template.json`.

So how do you run this script?